retrieve_token_balance.go

```diff
@@ -16,19 +16,22 @@ type RetrieveTokenBalanceRequest struct {
 	WalletID string `json:"wallet_id"`
 }
 
+// RetrieveTokenBalanceResult struct ...
```

Comment:
// RetrieveTokenBalanceResult fields that describes result from retrieve token balance. 
smth like this